package.json

```diff
@@ -35,13 +35,13 @@
         "react-native": "*"
     },
     "codegenConfig": {
-        "libraries": [
-            {
-                "name": "RNCWebViewSpec",
-                "type": "components",
-                "jsSrcsDir": "src"
-            }
-        ]
+        "name": "RNCWebViewSpec",
+        "type": "all",
+        "jsSrcsDir": "src",
+        "android": {
+            "javaPackageName": "com.reactnativecommunity.webview"
+        }
+
```

Comment:
This is the new syntax, from 0.70.0-rc.2.
It allows to have a single place where we define the specs.

the type "all" is required because we are generating both a module and a component.